ecr-login/cli/docker-credential-ecr-login/main.go

```diff
@@ -14,13 +14,36 @@
 package main
 
 import (
+	"flag"
+	"fmt"
+	"os"
+
 	ecr "github.com/awslabs/amazon-ecr-credential-helper/ecr-login"
 	"github.com/awslabs/amazon-ecr-credential-helper/ecr-login/api"
 	"github.com/awslabs/amazon-ecr-credential-helper/ecr-login/config"
+	"github.com/awslabs/amazon-ecr-credential-helper/ecr-login/version"
 	log "github.com/cihub/seelog"
 	"github.com/docker/docker-credential-helpers/credentials"
 )
 
+var vrsn bool
+
+const BANNER = `amazon-ecr-credential-helper
```

Comment:
Can you rename this to `banner`?  This constant doesn't need to be exported (start with a capital letter), and the Go convention of [Mixed Caps](https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps) applies to constants as well.

Reply:
Fixed by 527b0da